.github/workflows/test.yml

```diff
@@ -26,23 +26,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ckan-version: ['2.10']
-        solr-version: ['8']
+        ckan-version: ['2.11', '2.10']
```

Comment:
I disagree. With act (GitHub local action running) you can use these workflows locally.

For your own usage. You should have defaults if not set instead.

Reply:
k, tho we may need to look at using alt sources for container due to https://news.ycombinator.com/item?id=43125089